XML3-Homework/parrish_1-26_xml3.xml

```diff
@@ -0,0 +1,115 @@
+                right needle back and place on top of the left needle, bringing the loop of yarn
+                with it. You will have a loop on each needle. </knitStep>
+        </slipKnotSteps>
+        <slipKnotSteps slipKnot="eighth"> 8 <knitStep stitchStep="third"> How to knit stitch step 3
+                Now, transfer the loop on your right needle onto your left needle, so you have 2
+                loops on your left. You’ve cast on your first stitch! </knitStep>
+        </slipKnotSteps>
+        <slipKnotSteps slipKnot="ninth"> 9 Now repeat step 5 - 8 until you’ve cast on the amount of
+            stitches you’d like to knit! </slipKnotSteps>
+    </slipKnot>
+    <mainSteps>
+        <step main="fifth"> 5. Knit a row of stitches</step> Now you’ve cast on you’re ready to knit
+        your first row of stitches. The knit stitch, which is also called garter stitch, is the
+        first basic knitting stitch to master when you begin. We’ll show you how step-by-step. </mainSteps>
+
+    <knitStitchSteps> How to knit stitch <stitchSteps step="first"> 1 Insert your right needle
+            underneath the first stitch on your left needle. </stitchSteps>
+        <stitchSteps step="second"> 2 How to knit step 2 Bring the working yarn around the right
+            needle. </stitchSteps>
+        <stitchSteps step="third"> 3 How to knit step 3 Slide the right needle back bringing the
+            yarn with it, so the right needle sits on top of the left. </stitchSteps>
+        <stitchSteps step="fourth"> 4 How to knit step 4 Slide the stitch off the left hand needle
+            so you now have one knitted stitch on your right. You’ve completed your first knit
+            stitch. Go you! </stitchSteps>
+        <stitchSteps step="fifth"> 5 Knit your first row Repeat your knit stitch until you’ve
+            completed the first row! </stitchSteps>
+        <stitchSteps step="sixth"> 6 Turn your right needle around and transfer it to your left
+            hand. Swap your empty left hand needle into your right hand to begin stitching your
+            second row using steps 1-5. </stitchSteps>
+        <stitchSteps step="seventh"> 7 Repeat the number of rows you’d like to build your fabric.
+        </stitchSteps>
+    </knitStitchSteps>
+
+    <mainSteps>
+        <step main="sixth">6. Bind off</step> Binding off is the final knitting technique you’ll
+        need to complete your first knitting project. While there are lots of ways to bind off,
+        we’re going to show you a simple bind off method to get you started. </mainSteps>
+
+    <bindOffSteps> To ‘bind off’ just means fastening up your live stitches using your needles, so
+        you have a nice neat and secure edge. Once you’ve got this finishing technique in the bag
+        you're ready to start your first project! <header>How to make a simple bind off</header>
+        <bindOff step="first"> 1 How to cast off step 1 Knit your first stitch, as if you were
```

Comment:
@kzp308 Usually attributes that number things will hold just a simple number. You could actually remove the "1" from the text and pop it into the attribute value to hold that information directly.: `<bindoff step="1">How to cast...</bindoff>`.